docs/02-app/02-api-reference/04-functions/generate-viewport.mdx

```diff
@@ -61,7 +61,6 @@ export function generateViewport({ params }) {
 > **Good to know**:
 >
 > - If the viewport doesn't depend on runtime information, it should be defined using the static [`viewport` object](#the-viewport-object) rather than `generateMetadata`.
-> - Marking `generateViewport` as `async` is not yet supported. If you still require this functionality, continue using the previous `generateMetadata` option. We will be adding support for `async` in an upcoming minor release.
```

Comment:
Is it allowed now to be async?

Reply:
I would also like to know this. I created #57816 because the recommendation to use `generateMetadata` was incorrect. Removing the entire note indicates that either `async` is now supported or that the plans to add this support in a minor release were abandoned.